doc/nrf/ug_nrf52.rst

```diff
@@ -131,6 +138,41 @@ The nRF52 Series devices support running another protocol in parallel with the n
 The :ref:`Multi-Protocol Service Layer (MPSL) <nrfxlib:mpsl>` library provides services for multi-protocol applications.
 
 
+.. |note| replace:: There is currently no support for upgrading the upgradable bootloader (:doc:`mcuboot:index`) on nRF52 Series devices.
+
+.. fota_upgrades_start
+
+FOTA upgrades
+*************
+
+|fota_upgrades_def|
+FOTA upgrades can be used to replace the application.
+
+.. note::
+   |note|
+
+A FOTA upgrade requires the following steps:
+
+1. Make sure that your application supports FOTA upgrades.
+      To download and apply FOTA upgrades, the following requirements apply:
+
+      * Your application must enable :option:`CONFIG_MCUMGR_SMP_BT`.
+      * |fota_upgrades_req_mcuboot|
+
+#. Create a binary file that contains the new image.
+      |fota_upgrades_building|
+      The :file:`app_update.bin` file is the file that should be downloaded to the device.
```

Comment:
Should we note here that this file is only generated when `CONFIG_BOOTLOADER_MCUBOOT=y` ?

Reply:
We do - that's in the variable above ;)
"To create a binary file for an application upgrade, make sure that CONFIG_BOOTLOADER_MCUBOOT is enabled and build the application as usual. The build will create several binary files (see Using MCUboot in nRF Connect SDK). "

Reply:
Lol, I guess my built in rst preprocessor needs a patch :)